lib/repo_tools.pm

```diff
@@ -92,6 +93,13 @@ sub smt_mirror_repo {
     save_screenshot;
 }
 
+sub password_twice {
+    type_password;
+    send_key "tab";
+    type_password;
+    send_key "alt-o";
+}
+
 sub rmt_wizard {
     my $SCC_Password = get_var('SCC_PWD');
```

Comment:
We can remove these variables as you don't use them anymore.

Reply:
Yes, removed.